packages/showcases/core/csf/components/actions/button.csf.js

```diff
@@ -37,6 +37,27 @@ export const argTypes = {
       options: ['small', 'medium', 'large', 'stretched'],
     },
   },
+  type: {
+    type: { name: 'string', required: false },
+    description: 'The type of the button.',
+    defaultValue: 'button',
+    control: {
+      type: 'select',
+      options: ['button', 'submit'],
+    },
+  },
+  isDisabled: {
+    type: { name: 'boolean', required: false },
+    description: 'Whether the button is disabled.',
+    table: {
+      type: {
+        summary: 'boolean',
+      },
+      defaultValue: { summary: 'false' },
+    },
+    defaultValue: false,
+    control: { type: 'boolean' },
+  },
```

Comment:
Can you please rename it to `disabled` instead of `isDisabled`, we have this convention for all other components :)